### PR TITLE
Fix default liked, shuffle, etc. icons to more recognizable symbols

### PR DESCRIPTION
**Pull request by rasmuspeders1**
Commit [092165](https://github.com/Rigellute/spotify-tui/commit/092165e58a5c8fc4226c75b5d84f34bf298feaa6 ) broke the liked icon for me.

This pull request sets the default liked icon, along with the other shuffle,play etc. icons, to something more universally recognizable.

### CODE REVIEW

**Comment by rasmuspeders1** [comment]
maybe the black versions of play and pause are better?
▶
⏸

**Comment by rasmuspeders1** [comment]
Updated to use symbols listed here: https://en.wikipedia.org/wiki/Media_control_symbols

File: README.md [hunk 1]
```diff
@@ -248,12 +248,12 @@ behavior:
   # Determines the text icon to display next to "liked" Spotify items, such as
   # liked songs and albums, or followed artists. Can be any length string.
   # These icons require a patched nerd font.
```
Comment (Rigellute) [inline README.md:250]: Looks like the previous icons required a patched nerd font, which I must have overlooked

**Review by Rigellute — APPROVED** [review]
Thank you @rasmuspeders1 👍

**Comment by Rigellute** [comment]
@all-contributors please add @rasmuspeders1 for code

**Pull request merged**